pkg/volume/hostpath/host_path.go

```diff
@@ -305,6 +305,21 @@ func (r *hostPathProvisioner) Provision(selectedNode *v1.Node, allowedTopologies
 			},
 		},
 	}
+
+	var labels map[string]string
+	if r.options.PVC.Spec.Selector != nil {
+		labels = r.options.PVC.Spec.Selector.MatchLabels
+	}
+
+	if len(labels) != 0 {
+		if pv.Labels == nil {
+			pv.Labels = make(map[string]string)
+		}
+		for k, v := range labels {
+			pv.Labels[k] = v
```

Comment:
what if pv existing labels conflict with the ones from pvc?

Reply:
pvc would not be bound to pv, it is same with the current logic

Reply:
> https://github.com/kubernetes/kubernetes/pull/82424/files

No, we don't check if PVC and PV labels are the same during binding.

Reply:
> > https://github.com/kubernetes/kubernetes/pull/82424/files
> 
> No, we don't check if PVC and PV labels are the same during binding.



> > https://github.com/kubernetes/kubernetes/pull/82424/files
> 
> No, we don't check if PVC and PV labels are the same during binding.

Sorry, my mistake,.

 I thinks pv should match the `pvc selector` if a pv bound to pvc. but though we use the selector of pvc, pv dynamically created also does not the the matched label.